time/example_time_tuple.py

```diff
@@ -0,0 +1,10 @@
+import time
+
+ts=0
```

Comment:
Sorry for backlogging with merging this. Unfortunately, there're multiple PEP8 inconsistencies in this example, so I have to request to fix them.